src/partners/totle.ts

```diff
@@ -29,6 +29,19 @@ const asSwapCollectionResult = asArray(
   asObject({ returnValues: asObject({ id: asString }) })
 )
 
+const asSwapEventsResult = asArray(
+  asObject({
```

Comment:
Should make a separate cleaner per object in the array and clean each event inside the for loop around line 343